src/custom/hooks/useApproveCallback/index.ts

```diff
@@ -97,3 +97,41 @@ export function useApproveCallbackFromClaim({
     amountToCheckAgainstAllowance: approveAmounts?.amountToCheckAgainstAllowance,
   })
 }
+
```

Comment:
why is this needed, do the other hooks not suffice for what we want to do re: approving?

Reply:
As far as I can see one expects a claim and other a order object, so I think it makes sense to ad this one where we pass the balance